### DISCUSSION TITLE
CUAL ES EL PROCEDIMIENTO PARA SUBIR LAS KATAS  Y PODERLAS ENIAR

### DISCUSSION THREAD

**Discussion by RaulGuereca** [LaunchX-InnovaccionVirtual/CursoIntroPython#272] [General]
Me podrian decir cual es el procedimiento para enviar las Katas o donde puedo ver el video de las sesion del viernes 11 feb .
No se como entregar los archivos de las katas  para luego llenar el formato que estan solicitando para enviarlas.  Agradezco su ayuda.

**Comment by Giron1098** [comment]
Hola, publicaron un anuncio aquí en Git sobre cómo hacerlo. Te dejo el enlace, espero te sirva :)
https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/232

**Comment by AndresAVargas** [comment]
hay que entregar las 11 katas? o solo las que podamos hacer :C

**Reply by OscarIslasH** [reply]
Segun tengo entendido dijo @FernandaOchoa que daba igual si entregabamos 2 o 10 y esto que estamos haciendo ahora son practicas para que vayamos acostumbrandonos con terminos y como se programa. 

Ya si me equivoco pues entendi mal yo pero es lo que recuerdo del ultimo stream

La fecha de entrega es hasta el lunes 21

**Reply by Gracesl** [reply]
Estor nerviosa en este viernes voy en la kata 8, me pregunto si ya la mayoría entregó.

**Reply by gersondlarosa** [reply]
Osea que si tengo 7 katas con eso puedo avanzar al siguiente modulo ??? @FernandaOchoa

**Reply by FernandoTorillo** [reply]
Yo llevo 7 katas, me cuesta ya que vengo desde 0, pero creo entregar solo 7 :$ aunque, creo si dijeron que no importaba cuantas entregaras, un saludo
